lms/templates/registration/activate_account_notice.html

```diff
@@ -6,12 +6,16 @@
 <div class="wrapper-msg urgency-high">
     <div class="msg">
         <div class="msg-content">
-            <h2 class="title">${_("Thanks for Registering!")}</h2>
+            <h2 class="title">${_("You're almost there!")}</h2>
             <div class="copy">
                 <p class='activation-message'>${Text(_(
-                        "You've successfully created an account on {platform_name}. We've sent an account "
-                        "activation message to {email_start}{email}{email_end}. To activate your account and start enrolling in "
-                        "courses, click the link in the message."
+                        "There's just one more step: Before you
```

Comment:
To be clear here, the line length doesn't matter as long as there are beginning and ending double quotes on each line? (When I worked with this file earlier, I didn't realize that, for example, the original line 13 had wrapped in the text editor I was using, so although I thought it was odd to only have beginning and ending double quotes on alternate lines I thought I was doing the right thing... Aargh.)